clarification on minidumps docs (#13732)
getsentry/sentry-docs: docs/platforms/native/guides/minidumps/index.mdx

they might contain sensitive information on the target system, such as
environment variables, local pathnames or maybe even in-memory representations
of input fields, including passwords. **Sentry does not store these memory

> dumps by default**. Once processed, they are removed immediately, and all sensitive
> information is stripped from the resulting issues.

Author: ```suggestion
dumps by default**. Once processed, they are removed immediately, and all sensitive
information is stripped from the resulting issues. 
You have the option to configure Sentry to retain the minidump as an attachment.
```

they might contain sensitive information on the target system, such as
environment variables, local pathnames or maybe even in-memory representations
of input fields, including passwords. **Sentry does not store these memory
dumps by default**. Once processed, they are removed immediately, and all sensitive
information is stripped from the resulting issues. 

> You have the option to configure Sentry to retain the minidump as an attachment.

Author: ```suggestion
You can configure Sentry to store the minidump as attachment.
```
